src/main/webapp/swagger-ui/index.html

```diff
@@ -1,166 +1,157 @@
+            return urlMatches[1];
+          },
+          securityConfig: function (cb) {
+            $.getJSON(this.baseUrl() + '/swagger-resources/configuration/security', function (data) {
+              cb(data);
             });
-
-            function maybePrefix(location, withRelativePath) {
-                var pat = /^https?:\/\//i;
-                if (pat.test(location)) {
-                    return location;
-                }
-                return withRelativePath + location;
-            }
-
-            function initializeBaseUrl() {
-                var relativeLocation = springfox.baseUrl();
-
-                $('#input_baseUrl').hide();
-
-                $.getJSON(relativeLocation + "/swagger-resources", function(data) {
-
-                    var $urlDropdown = $('#select_baseUrl');
-                    $urlDropdown.empty();
-                    $.each(data, function(i, resource) {
-                        var option = $('<option></option>')
-                        .attr("value", maybePrefix(resource.location, relativeLocation))
-                        .text(resource.name + " (" + resource.location + ")");
-                        $urlDropdown.append(option);
-                    });
-                    $urlDropdown.change();
-                });
-
+          },
+          uiConfig: function (cb) {
+            $.getJSON(this.baseUrl() + '/swagger-resources/configuration/ui', function (data) {
+              cb(data);
+            });
+          },
+        };
+        window.springfox = springfox;
+        window.oAuthRedirectUrl = springfox.baseUrl() + './dist/o2c.html';
+
+        window.springfox.uiConfig(function (data) {
+          window.swaggerUi = new SwaggerUi({
+            dom_id: 'swagger-ui-container',
+            validatorUrl: data.validatorUrl,
+            supportedSubmitMethods: ['get', 'post', 'put', 'delete', 'patch'],
+            onComplete: function (swaggerApi, swaggerUi) {
+              initializeSpringfox();
+              if (window.SwaggerTranslator) {
+                window.SwaggerTranslator.translate();
+              }
+              $('pre code').each(function (i, e) {
+                hljs.highlightBlock(e);
+              });
+            },
+            onFailure: function (data) {
+              log('Unable to Load SwaggerUI');
+            },
+            docExpansion: 'none',
+            apisSorter: 'alpha',
+            showRequestHeaders: false,
+          });
+
+          initializeBaseUrl();
+
+          $('#select_baseUrl').change(function () {
+            window.swaggerUi.headerView.trigger('update-swagger-ui', {
+              url: $('#select_baseUrl').val(),
+            });
+            addApiKeyAuthorization();
+          });
+
+          function addApiKeyAuthorization() {
+            var authToken = JSON.parse(localStorage.getItem('accessToken') || sessionStorage.getItem('accessToken'));
```

Comment:
why this file has so much diff?
can I assume you change this content?
others are indention problem?

Reply:
I think the changed is made by my VS code formatter. Because what I did was just changed the jhi-authenticationToken to accessToken only